scania-digital-design-system/sdds
# fix: standardise Storybook controls

**Describe pull-request**  
An attempt to standardise the Storybook controls in all available components. Also updated control descriptions, added default values to controls and re-structured deprecated control types. Also some minor various fixes that were found along the way.  

**Solving issue**  
Fixes: [AB#2696](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2696)

**How to test**  
_Add description how to test if possible_
1. Go to Storybook through the link below.
2. Check order of controls and descriptions of all components.

**Suggested test steps**

**Screenshots**  


**Additional context**
